src/main/scripts/install.sh

```diff
@@ -25,27 +25,47 @@ while getopts "l:u:p:" opt; do
         p)
             password=$OPTARG #password of IBM user id for downloading artifacts from IBM web site
         ;;
+        t)
+            wasNDTraditional=$OPTARG #IBM WebSphere Application Server ND Traditional version
+        ;;
+        j)
+            ibmJavaSDK=$OPTARG #IBM Java SDK version
+        ;;        
     esac
 done
 
 # Variables
-imKitName=agent.installer.linux.gtk.x86_64_1.9.0.20190715_0328.zip
-repositoryUrl=http://www.ibm.com/software/repositorymanager/com.ibm.websphere.ND.v90
-wasNDTraditional=com.ibm.websphere.ND.v90_9.0.5001.20190828_0616
-ibmJavaSDK=com.ibm.java.jdk.v8_8.0.5040.20190808_0919
+SSLPREF="com.ibm.cic.common.core.preferences.ssl.nonsecureMode=false"
```

Comment:
I think we should move these variables to a single file (virtualimage.properties) and source this file in install.sh and was-check.sh:

SSLPREF="com.ibm.cic.common.core.preferences.ssl.nonsecureMode=false"
DOWNLOADPREF="com.ibm.cic.common.core.preferences.preserveDownloadedArtifacts=false"
imKitName=agent.installer.linux.gtk.x86_64_1.9.zip
repositoryUrl=https://www.ibm.com/software/repositorymanager/entitled
wasNDTraditional=com.ibm.websphere.ND.v90_9.0.5007.20210301_1241
ibmJavaSDK=com.ibm.java.jdk.v8_8.0.6026.20210226_0840

We should also add install directories in this file:
/datadrive/IBM/InstallationManager/V1.9
/datadrive/IBM/WebSphere/ND/V9
/datadrive/IBM/IMShared

This file will be a single place which we/customer can refer to to get details about the installation.

Reply:
Good suggestion. Fixed in [this commit](https://github.com/WASdev/azure.websphere-traditional.image/pull/13/commits/04aaf3779f49a62911eff5f114ba0f31d1c1748b).